mobs/nakki_v2/NakkiV2.cs

```diff
@@ -349,6 +353,10 @@ public void PlayLilypadAttackAnimation() {
 		AnimationPlayer.Play("lilypad_attack");
 	}
 
+	public async Task PlayLilypadAttackAnimationAsync(CancellationToken? ct = null) {
+		await AnimationPlayer.PlayAsync("lilypad_attack", ct);
+	}
```

Comment:
Passing the `CancellationToken` isn't strictly necessary here, as the animation changes should cancel the task anyway, but added it in here just in case. It's also consistently ugly this way, having the CTs around all the asyncs.